kubernetes.go

```diff
@@ -548,6 +558,53 @@ func (svc *KubernetesServiceOp) Delete(ctx context.Context, clusterID string) (*
 	return resp, nil
 }
 
+// DeleteSelective deletes a Kubernetes cluster and the specified associated resources.
```

Comment:
I like the comments you've left on these functions. Clear and to the point 👍

Reply:
Same!